commands/version.go

```diff
@@ -0,0 +1,24 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version number of Cartridge CLI",
+	Long:  `All software has versions. This is Cartridge CLI's`,
+	Args:  cobra.MaximumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		initLogger()
+
+		// TODO: improve getting version
+		fmt.Println("Cartridge CLI v2.0.0")
```

Comment:
As preparation step you could move "2.0.0" into global variable that could be redefined  at compile time.

Reply:
Yes, I will improve getting version in a separated PR.